src/server.prod.js

```diff
@@ -82,7 +82,8 @@ class ProdServer {
 
   getRenderFunction() {
     const { serverBundle, clientManifest, template } = this.readBundleAndManifest();
-    const bundleOptions = { ...renderOptions, template, clientManifest };
+    const { bundleRendererOptions: configRenderOptions = {} } = this.config;
```

Comment:
why the rename?

Reply:
To align the var name to the name `renderOptions`. Just to make clear where these options come from.